packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -172,6 +172,10 @@ export default class AgoraIpOrderCtrl {
             isIpBlockOffer: maximumQuantity === 1,
           };
         });
+        console.log('bo1', bo);
```

Comment:
**nitpick:** Please remove the console statements.

Reply:
removed it